FFmpegInterop/Source/MediaSampleProvider.cpp

```diff
@@ -100,7 +111,15 @@ MediaStreamSample^ MediaSampleProvider::GetNextSample()
 		// Write the packet out
 		hr = WriteAVPacketToStream(dataWriter, &avPacket);
 
-		Windows::Foundation::TimeSpan pts = { LONGLONG(av_q2d(m_pAvFormatCtx->streams[m_streamIndex]->time_base) * 10000000 * avPacket.pts) };
+		if (StartOffset == 0) {
```

Comment:
Won't this cause the PTS to be off by one frame? If the start offset is actually 0, then we'll set the StartOffset as the next timestamp found. Should StartOffset be initially set to a negative value to allow for the value of 0 to be valid?